test/test_mps.py

```diff
@@ -1482,6 +1482,11 @@ def _linear_helper(self, in_features, out_features, shape, bias=True, backward_p
                 self.assertEqual(cpu_linear.bias.grad.size(), mps_linear.bias.grad.size())
                 self.assertEqual(cpu_linear.bias.grad, mps_linear.bias.grad.to("cpu"), atol=8e-04, rtol=10.4e-05)
 
+            inputs = (linear_mps_input, mps_linear.weight)
+            if bias:
+                inputs += (mps_linear.bias,)
+            gradgradcheck(torch.nn.functional.linear, inputs, atol=gradgrad_atol, rtol=10.4e-05, raise_exception=True, fast_mode=False)
```

Comment:
I don't think you should be using gradcheck in fp32 tbh.
You might as well do the backward for a random set of gradient compared to CPU and make sure these match.
Even for a reduction op like this one, it should give you meaningfully similar results.

Another option is to check only the gradient (not double backprop) of `torch.ops.aten.linear_backward`.

Reply:
Thanks a lot! So would you suggest I remove the gradgradcheck at all or keep it as is?

Reply:
Yes we should remove the gradgradcheck test and all the tolerance stuff yes.

Reply:
Done, please take a look.